common/src/nostr/mod.rs

```diff
@@ -85,10 +87,11 @@ impl NostrClient {
         Ok(())
     }
 
-    pub async fn submit_trade_token_to_seller(
+    // client specific function?
```

Comment:
I think we can put these functions in the client as they are not used by the coordinator